### ISSUE TITLE
labels for dataset

### ISSUE THREAD

**Issue by Desolatexxx** [jaehyunnn/DeepAerialMatching_pytorch#4]
Hello, could you please provide a method for creating labels for your dataset?

**Comment by fps0513** [comment]
Hello, may I ask how you resolved it later?

**Comment by jaehyunnn** [comment]
Please refer to the `_generate_random_affine` method in `https://github.com/jaehyunnn/DeepAerialMatching_pytorch/blob/master/src/data/train_dataset.py`
